prometheus_client/multiprocess.py

```diff
@@ -143,7 +150,7 @@ def _accumulate_metrics(metrics, accumulate):
                         samples[(metric.name + '_count', labels)] = acc
 
             # Convert to correct sample format.
-            metric.samples = [Sample(name_, dict(labels), value) for (name_, labels), value in samples.items()]
+            metric.samples = [Sample(name_, dict(labels), value, sample_timestamp_secs.get((name_, labels), None)) for (name_, labels), value in samples.items()]
```

Comment:
I don't think we should be exposing the timestamp in the final rendered result. The fact that we store the timestamp should be an internal detail only for use with mostrecent.

Reply:
OK. I can drop this from this PR. I have a usage on the timestamp part (actually, my usage is exactly the same as Ruby's PR, exposing GitHub API limits), but as you said in the other comment, this usage must be a rare case so need to be careful. Let me add that a FR separated from this PR.